packages/database/doc/sync_functions.md

```diff
@@ -0,0 +1,18 @@
+# Sync information
+
+The `sync_info` table is meant to always be accessed through one of these two functions: `propose_sync_task` and `end_sync_task`.
+This acts as a semaphore, so that two workers (e.g. the roam plugin on two different browsers) do not try to run the same sync task at the same time. So you need to give the function `propose_sync_task` enough information to distinguish what you mean to do:
+
+1. The `target`, e.g. the database Id of the scope of the task, usually a space, but it could be a single content or concept (for reactive updates)
+2.  a `function` name, to distinguish different tasks on the same target; e.g. adding vs deleting content. (arbitrary short string)
```

Comment:
@maparent how do we distinguish sync table calls for Discourse Node types (schema) vs Discourse Node instances (nodes)
    - is it `function`?  In the deleted docs it says this in an arbitrary string
    - eg: use case: Discourse Node types (schema) was last synced 1 hour ago but Discourse Node instances (nodes) was synced 10 seconds ago

Reply:
Yes, it is function.